Naming convention

asashour opened the pull request:
`static final` fields should be upper case, other fields and variables should not.

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/vaadin/framework/10242)
<!-- Reviewable:end -->

asashour committed:
Variable names to conform to naming convention.

asashour committed:
typo

asashour committed:
Fix tests

asashour committed:
Fix Profiler

pekam commented:
Reviewed 74 of 84 files at r1.
Review status: 72 of 87 files reviewed at latest revision, 4 unresolved discussions.

---

*[client/src/main/java/com/vaadin/client/VTooltip.java, line 474 at r1](https://reviewable.io:443/reviews/vaadin/framework/10242#-KxMcfuIwwd93PXYtyfN:-KxMcfuIwwd93PXYtyfO:b-yslag3) ([raw file](https://github.com/vaadin/framework/blob/5b9120792be1d772981169363b8a00461cd78720/client/src/main/java/com/vaadin/client/VTooltip.java#L474)):*
> ```Java
>     }
> 
>     private static final int EVENT_XY_POSITION_OUTSIDE = -5000;
> ```

Feels weird having these three variables declared in the middle of the class, especially since those are used in the methods above.
So maybe these should be moved to the top now when we are working on this?

---

*[client/src/main/java/com/vaadin/client/ui/VTabsheet.java, line 752 at r1](https://reviewable.io:443/reviews/vaadin/framework/10242#-KxMjW8ttcr4Iq4GDB83:-KxMjW8ttcr4Iq4GDB84:b-r6d36) ([raw file](https://github.com/vaadin/framework/blob/5b9120792be1d772981169363b8a00461cd78720/client/src/main/java/com/vaadin/client/ui/VTabsheet.java#L752)):*
> ```Java
>     int tabulatorIndex = 0;
> 
>     private static final FocusImpl FOCUS_IMPL = FocusImpl.getFocusImplForPanel();
> ```

Maybe declaration should be moved to the top?

---

*[client/src/main/java/com/vaadin/client/ui/richtextarea/VRichTextToolbar.java, line 278 at r1](https://reviewable.io:443/reviews/vaadin/framework/10242#-KxMl9XLx8TfEs5glIAm:-KxMl9XLx8TfEs5glIAn:b4ckmlv) ([raw file](https://github.com/vaadin/framework/blob/5b9120792be1d772981169363b8a00461cd78720/client/src/main/java/com/vaadin/client/ui/richtextarea/VRichTextToolbar.java#L278)):*
> ```Java
>     }
> 
>     private static final RichTextArea.FontSize[] FONT_SIZES_CONSTANTS = {
> ```

Declaration to the top?

---

*[testbench-api/src/main/java/com/vaadin/testbench/elements/AbstractComponentElement.java, line 38 at r1](https://reviewable.io:443/reviews/vaadin/framework/10242#-KxN6Eq1xrdlJ_o1_OW2:-KxN6Eq1xrdlJ_o1_OW3:b-j07kid) ([raw file](https://github.com/vaadin/framework/blob/5b9120792be1d772981169363b8a00461cd78720/testbench-api/src/main/java/com/vaadin/testbench/elements/AbstractComponentElement.java#L38)):*
> ```Java
>      */
>     public String getCaption() {
>         final String idAttribute = "aria-labelledby";
> ```

Why is gwt dropped from the name?

And why is this variable kept in the method but the other one below is extracted as a static constant?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/framework/10242)*
<!-- Sent from Reviewable.io -->

asashour committed:
Field location

asashour commented:
Review status: 72 of 87 files reviewed at latest revision, 4 unresolved discussions, some commit checks failed.

---

*[client/src/main/java/com/vaadin/client/VTooltip.java, line 474 at r1](https://reviewable.io:443/reviews/vaadin/framework/10242#-KxMcfuIwwd93PXYtyfN:-KxNB6lAcejpPsWAktwN:b-896fix) ([raw file](https://github.com/vaadin/framework/blob/5b9120792be1d772981169363b8a00461cd78720/client/src/main/java/com/vaadin/client/VTooltip.java#L474)):*
<details><summary><i>Previously, pekam (Pekka Maanpää) wrote…</i></summary><blockquote>

Feels weird having these three variables declared in the middle of the class, especially since those are used in the methods above.
So maybe these should be moved to the top now when we are working on this?
</blockquote></details>

Done.

---

*[client/src/main/java/com/vaadin/client/ui/VTabsheet.java, line 752 at r1](https://reviewable.io:443/reviews/vaadin/framework/10242#-KxMjW8ttcr4Iq4GDB83:-KxNBQ4Pr5a5EsUx-nM9:b-3wmadx) ([raw file](https://github.com/vaadin/framework/blob/5b9120792be1d772981169363b8a00461cd78720/client/src/main/java/com/vaadin/client/ui/VTabsheet.java#L752)):*
<details><summary><i>Previously, pekam (Pekka Maanpää) wrote…</i></summary><blockquote>

Maybe declaration should be moved to the top?
</blockquote></details>

Moved to the nearest group of `static` fields.

---

*[client/src/main/java/com/vaadin/client/ui/richtextarea/VRichTextToolbar.java, line 278 at r1](https://reviewable.io:443/reviews/vaadin/framework/10242#-KxMl9XLx8TfEs5glIAm:-KxN9Yzaey-MAARBi0PX:b24d5so) ([raw file](https://github.com/vaadin/framework/blob/5b9120792be1d772981169363b8a00461cd78720/client/src/main/java/com/vaadin/client/ui/richtextarea/VRichTextToolbar.java#L278)):*
<details><summary><i>Previously, pekam (Pekka Maanpää) wrote…</i></summary><blockquote>

Declaration to the top?
</blockquote></details>

It seems the general convention is to have inner interfaces and classes on top, then comes the fields. In this case, there are other fields, e.g. `images`, `strings`, etc.

I can put the inner types at the bottom, but I guess it should be done in a separate PR, since other places as well will mostly be changed.

---

*[testbench-api/src/main/java/com/vaadin/testbench/elements/AbstractComponentElement.java, line 38 at r1](https://reviewable.io:443/reviews/vaadin/framework/10242#-KxN6Eq1xrdlJ_o1_OW2:-KxNAeEnW-317TahQHew:b-e6vxfc) ([raw file](https://github.com/vaadin/framework/blob/5b9120792be1d772981169363b8a00461cd78720/testbench-api/src/main/java/com/vaadin/testbench/elements/AbstractComponentElement.java#L38)):*
<details><summary><i>Previously, pekam (Pekka Maanpää) wrote…</i></summary><blockquote>

Why is gwt dropped from the name?

And why is this variable kept in the method but the other one below is extracted as a static constant?
</blockquote></details>

`gwt` was dropped because I believe it makes add a value, anyhow it is now reverted. It is a variable because it is used 3 times.

The other variable is now removed, and directly used.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/framework/10242)*
<!-- Sent from Reviewable.io -->

asashour committed:
Merge branch 'master' into namingConvention

asashour committed:
Merge branch 'master' into namingConvention

pekam commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

---

Reviewed 10 of 84 files at r1, 1 of 1 files at r2, 3 of 3 files at r3, 1 of 1 files at r4, 3 of 3 files at r5, 1 of 1 files at r6.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/framework/10242#-:-KxNL5oH8wT-mbvnin4Y:bnfp4nl)*
<!-- Sent from Reviewable.io -->
